test/prow/prow.go

```diff
@@ -108,6 +108,11 @@ type Finished struct {
 // Metadata contains metadata in finished.json
 type Metadata map[string]interface{}
 
+// IsCI returns whether the current environment is a CI environment.
```

Comment:
Now, do we need this function anymore?

Reply:
We still need it. As if we have a test flow that can be run both locally and in the CI environment, we need to use this function to decide whether we should call `GetEnvConfig` to get the env vars set by Prow.

Reply:
So it'll be used in future code?

Reply:
Yes it will. More specifically, in the current shell script we have an environment variable called `IS_PROW`, later on when we rewrite the shell scripts into Go, we will replace that env var with this function.